File: C/menor_numero.c
#include <stdio.h>

float menor_numero(float numero[], int tamanho_vetor);

int main()
{
    printf("MENOR NUMERO\n\n");

    float numero[5];
    char nomes_numero[5][10] = {"primeiro","segundo","terceiro","quarto","quinto"};
    int tamanho_vetor = sizeof(numero) / sizeof(numero[0]);

    for (int i = 0; i < tamanho_vetor; i++)
    {
        printf("Digite o %s numero: ", nomes_numero[i]);
        scanf("%f", &numero[i]);
    }

    printf("\nMenor numero: %.1f", menor_numero(numero, tamanho_vetor));
    
    return 0;
}

float menor_numero(float numero[], int tamanho_vetor)
{
    //calcula o menor número.
    int menor = numero[0];
    for (int i = 1; i < tamanho_vetor; i++)
    {
        if (numero[i] < menor)
            menor = numero[i];
    }
    return menor;
}

File: Python/menor_numero.py
def menor_numero(numero):
    menor = numero[0]
    for i in range(len(numero)):
        if numero[i] < menor:
            menor = numero[i]
    return menor

def men_numero(numero):
    return min(numero)

print("MENOR NÚMERO\n")

nome_numero = ["primeiro","segundo","terceiro"]
numero = []

for i in range(3):
    numero.append(int(input(f"Digite o {nome_numero[i]} número: ")))

print(f"\nMenor número: {menor_numero(numero)} | {men_numero(numero)}")